acceptance/tests/utf8/utf8-in-catalog.rb

```diff
@@ -1,6 +1,6 @@
 test_name 'utf-8 characters in cached catalog' do
 
-  tag 'audit:high', # utf-8 is high impact in general
+  tag 'audit:medium',
```

Comment:
I'd like to continue testing this and `utf8-in-file-resource.rb` and `utf8-recursive-copy.rb` below, as this an area that is easy to regress due to variations in ruby, linux vs mac vs windows.

Reply:
Thank you @joshcooper . I have pushed an amended commit to restore these three tests.